main.go

```diff
@@ -963,6 +964,11 @@ corectl eval by "Region" // Returns the values for dimension "Region"`,
 
 func init() {
 
+	// If running a released version then check if there is a later version
+	if version != "" {
+		checkLatestVersion()
```

Comment:
I'm not too fond of us doing this call every execution of corectl. What happens if you are offline or on a slow network, will it block execution?

Reply:
I have added a check for if the github request failed. Seems to cause no obvious delay when being offline. Slow network however might be harder to verify.

Reply:
If we run it on a separate thread it might not be the first print in the log. Looks kind of weird imo, but maybe not a major issue.